crates/typstyle-core/src/attr.rs

```diff
@@ -171,6 +171,10 @@ impl AttrStore {
         if commented {
             self.set_commented(node);
         }
+        if commented && node.cast::<ModuleImport>().is_some() {
+            // no format module import if it has comments inside
+            self.set_format_disabled(node);
+        }
```

Comment:
This is unfortunate because i didn't come up with a good method to handle comments here